### ISSUE TITLE
Mod description is always written in the English description page in Steam.

### ISSUE THREAD

**Issue by Randy624** [Regalis11/Barotrauma#11105]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

In v0.21.1.0 you fix mod descriptions being truncated to 255 characters.
Can you also fix the language issue here. The description is always written in the English description in Steam. (need to be written in your language)

### Reproduction steps

Open excisting mod (in game menu)
Publish it
When opened, a description in your language is loaded. When publishing - it is written in the English description on Steam

### Bug prevalence

Happens every time I play

### Version

0.20.16.1

### Which operating system did you encounter this bug on?

Windows

**Comment by BiffW** [comment]
I'd reword this like this: when you update your mod with your language(except English), it always overwrites English decription.

**Comment by Regalis11** [comment]
Fixed in https://github.com/Regalis11/Barotrauma-development/commit/8dbc23fd1a795e2def31698f5e38409b6016bb6e
